daily-challenges/2024-11/28/lcaloi.29.spec.ts

```
@@ -0,0 +1,38 @@
+        }
+        else if (loopCount >= 5) {
+            throw new Error(`ERROR: get target pokemon failed!!! `);
+        }
+        loopCount++;
+    } while (!targetName);
+
+    const arrLocator = await page.locator(`div.bag`).all();
+
+    for (const element of arrLocator) {
+        await element.click();
+        let pokemonName = await element.locator('div.pokemon-name').textContent();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'pokemonName' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)